site2/website/versioned_docs/version-2.10.x/client-libraries-cpp.md

```diff
@@ -654,7 +654,7 @@ Client client("pulsar+ssl://my-broker.com:6651", config);
 
 ```
 
-For complete examples, refer to [C++ client examples](https://github.com/apache/pulsar/tree/master/pulsar-client-cpp/examples).
+For complete examples, refer to [C++ client examples](https://github.com/apache/pulsar-client-cpp/examples).
```

Comment:
this url needs to be fixed. please check that subdirectories are properly referenced.

Reply:
it seems that `main` is the default branch now. so this url is https://github.com/apache/pulsar-client-cpp/tree/main/examples

Reply:
Great catch. It should be https://github.com/apache/pulsar-client-cpp/tree/main/examples

Reply:
Fixed